Samples/ReVerC/Operation.qs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Quantum.ReVerC
```

Comment:
The existing samples have been placed under the `Microsoft.Quantum.Samples` namespace, would you be willing to rename the namespace to `Microsoft.Quantum.Samples.ReVerC`? Thanks!

Reply:
ReVerC compiles to the namespace `Quantum.ReVerC`, so barring a change on that end (which is certainly possible), Adder.qs will use the namespace `Quantum.ReVerC`.

A simple solution would be for both Driver and Operation to use the namespace `Microsoft.Quantum.Samples.ReVerC` and open `Quantum.ReVerC`. The C# compiler seems to be doing that fine, but the Q# compiler seems to think `Quantum.ReVerC` is a sub-namespace of `Microsoft.Quantum` and isn't able to import it.

As it is I've left Driver using the `Microsoft.Quantum.Samples.ReVerC` namespace, and Operation and Adder using `Quantum.ReVerC`. Let me know if that's not ideal and I can push an update to ReVerC!